### PR TITLE
get.R: adding assertions

### PR DESCRIPTION
**Pull request by grewered**
- [ ] get.plausible
- [ ] get.wle
- [ ] get.shw
- [ ] get.prm
- [ ] get.itn
- [ ] get.dsc
- [ ] get.equ

### CODE REVIEW

File: R/get.R [hunk 1]
```diff
@@ -4,14 +4,19 @@
 
 ### called by getConquestPVs ---------------------------------------------------
 
-get.plausible <- function(file, quiet = FALSE, forConquestResults = FALSE)  {
+get.plausible <- function(file, quiet = FALSE, forConquestResults = FALSE){
+  checkmate::assert_file(file)
```
Comment (grewered) [inline R/get.R:8]: Should I use `assert_file` or `assert_character` for asserting the `file` argument?

File: R/get.R [hunk 2]
```diff
@@ -153,6 +177,9 @@ get.shw <- function(file, dif.term, split.dif = TRUE, abs.dif.bound = 0.6, sig.d
       results.sel<- data.frame(inp.sel,filename=as.character(file),stringsAsFactors = FALSE)
       sapply(intersect(c("ESTIMATE", "ERROR"), colnames(results.sel)), FUN = function (colx) {if(any(is.na(as.numeric(results.sel[,colx])))) {cat(paste0("'",colx,"' column in Outputfile for term '",all.terms[length(all.terms)],"' in file: '",file,"' does not seem to be a numeric value. Please check!\n"))}})
       if(!missing(dif.term)) {                                    ### Der absolute DIF-Wert ist 2 * "Betrag des Gruppenunterschieds". Fuer DIF muessen ZWEI Kriterien erfuellt sein:
+        checkmate::assert_character(dif.term, len = 1)
+        ind2 <- grep(dif.term, input.all, fixed = TRUE)
+        assert_numeric(ind2, len = 1)
```
Comment (grewered) [inline R/get.R:182]: I added these line from the old pull request #4.

File: R/get.R (hunk 1 above)
Comment (weirichs) [reply R/get.R:8]: I think, `assert_file()` is appropriate here.

File: R/get.R [hunk 3]
```diff
@@ -153,6 +177,9 @@ get.shw <- function(file, dif.term, split.dif = TRUE, abs.dif.bound = 0.6, sig.d
       results.sel<- data.frame(inp.sel,filename=as.character(file),stringsAsFactors = FALSE)
       sapply(intersect(c("ESTIMATE", "ERROR"), colnames(results.sel)), FUN = function (colx) {if(any(is.na(as.numeric(results.sel[,colx])))) {cat(paste0("'",colx,"' column in Outputfile for term '",all.terms[length(all.terms)],"' in file: '",file,"' does not seem to be a numeric value. Please check!\n"))}})
       if(!missing(dif.term)) {                                    ### Der absolute DIF-Wert ist 2 * "Betrag des Gruppenunterschieds". Fuer DIF muessen ZWEI Kriterien erfuellt sein:
+        checkmate::assert_character(dif.term, len = 1)
```
Comment (weirichs) [inline R/get.R:180]: very well, thanks!

**Review by weirichs — APPROVED** [review]
thanks, LGTM

**Comment by grewered** [comment]
@weirichs 
can you look at the conflicts? it's in the function `get.dsc`, but I didn't make these changes (apart from assert_file). Do I use the old or the new code?

**Pull request merged**